### PR TITLE
update most of the MSDN links that were redirecting

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/localization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/develop/localization.md) | :white_check_mark:Succeeded | |
[docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md) | :white_check_mark:Succeeded | |
[docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md) | :white_check_mark:Succeeded | |
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808081618092634-301%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=S70gpKp21nOMUWWof%2F%2FzOBPeGedbvlwGn6ZpZ3HYGCQ%3D&st=2018-08-08T16%3A14%3A23Z&se=2018-09-08T16%3A19%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/localization.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ You can get the locale of the data of the hosting application by using the [cont
...
+You can globalize and use the [Date](http://msdn.microsoft.com/library/caf98d32-2de2-4704-8198-692350343681.aspx) and [Number](http://msdn.microsoft.com/library/c216d3a1-12ae-47d1-bca1-c3666d04572f.aspx) JavaScript type extensions and the JavaScript [Date](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Date) object in the JavaScript code for an Office Add-in to display values based on the locale settings on the current browser. For more information, see [Walkthrough: Globalizing a Date by Using Client Script](http://msdn.microsoft.com/library/69b34e6d-d590-4d03-a763-b7ae54b47d74.aspx).
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/develop/localization.md]: Yes. I only updated the ones that weren't going to msdn anymore. Most of the remaining msdn links still land on msdn.

File: docs/testing/attach-debugger-from-task-pane.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ For more information about debugging in Visual Studio, see the following:
 
 -	To launch and use the DOM Explorer in Visual Studio, see Tip 4 in the [Tips and Tricks](https://blogs.msdn.microsoft.com/officeapps/2013/04/16/building-great-looking-apps-for-office-using-the-new-project-templates/#tips_tricks) section of the [Building great-looking apps for Office using the new project templates](https://blogs.msdn.microsoft.com/officeapps/2013/04/16/building-great-looking-apps-for-office-using-the-new-project-templates) blog post.
 -	To set breakpoints, see [Using Breakpoints](https://msdn.microsoft.com/library/5557y8b4.aspx).
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/testing/attach-debugger-from-task-pane.md]: please see my other response

File: docs/develop/localization.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlexJerabek) [reply docs/develop/localization.md]: Cool. Not sure why I thought all of msdn was going down. Ignore those comments.

My earlier question about the developer account still stands.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/localization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/develop/localization.md) | :white_check_mark:Succeeded | |
[docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md) | :white_check_mark:Succeeded | |
[docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md) | :white_check_mark:Succeeded | |
[docs/publish/centralized-deployment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/publish/centralized-deployment.md) | :white_check_mark:Succeeded | |
[docs/testing/attach-debugger-from-task-pane.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/testing/attach-debugger-from-task-pane.md) | :white_check_mark:Succeeded | |
[docs/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-msdn-links-that-redirected/docs/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808081937571852-301%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=IhfznJLNOpr4bKjd8R3TYEPG4P94pVeOlA7aOd5pGPM%3D&st=2018-08-08T19%3A34%3A08Z&se=2018-09-08T19%3A39%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**